sdk/cognitivelanguage/ai-language-text/Samples.md

```diff
@@ -0,0 +1,764 @@
+
+const documents = [
+  "The WHO is issuing a warning about Monkey Pox.",
+  "Mo Salah plays in Liverpool FC in England.",
+];
+
+export async function main() {
+  const results = await client.analyze("DynamicClassification", documents, "en", {
+    categories: ["Health", "Politics", "Music", "Sports"],
+    classificationType: "Multi",
+  });
+
+  for (const result of results) {
+    console.log(`- Document ${result.id}`);
+    if (!result.error) {
+      console.log("\tClassifications:");
+      for (const category of result.classifications) {
+        console.log(`\t- ${category.category}`);
+      }
+    } else {
+      console.error("  Error:", result.error);
+    }
+  }
+}
+
+main();
+```
+
+### Healthcare Analysis
+
+Healthcare analysis identifies healthcare entities. For example, given input text "Prescribed 100mg ibuprofen, taken twice daily", the service returns "100mg" categorized as Dosage, "ibuprofen" as MedicationName, and "twice daily" as Frequency.
+
+FHIR, or Fast Healthcare Interoperability Resource, is a standard that defines how healthcare information can be exchanged by different computer systems. To receive a FHIR bundle for a particular text document, input `4.0.1` for the `fhirVersion`. You can also pass the `document_type` if the type of document in the source text is known. Supported document types include "ClinicalTrial", "DischargeSumamry", "ProgressNote", and more, which can be found in this [reference docs](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-ai-textanalytics/5.3.0b1/azure.ai.textanalytics.html#azure.ai.textanalytics.HealthcareDocumentType)
+
+```javascript
+const { AzureKeyCredential, TextAnalysisClient } = require("@azure/ai-language-text");
+
+const client = new TextAnalysisClient("<endpoint>", new AzureKeyCredential("<API key>"));
+
+const documents = [
+  "Prescribed 100mg ibuprofen, taken twice daily.",
+  "Patient does not suffer from high blood pressure.",
+];
+
+async function main() {
+  const actions = [
+    {
+      kind: "Healthcare",
+      fhirVersion: "4.0.1",
+      documentType: KnownHealthcareDocumentType.DischargeSummary,
+    },
+  ];
+  const poller = await client.beginAnalyzeBatch(actions, documents, "en");
+  const results = await poller.pollUntilDone();
+
+  for await (const actionResult of results) {
+    if (actionResult.kind !== "Healthcare") {
+      throw new Error(`Expected a healthcare results but got: ${actionResult.kind}`);
+    }
+    if (actionResult.error) {
+      const { code, message } = actionResult.error;
+      throw new Error(`Unexpected error (${code}): ${message}`);
+    }
+    for (const result of actionResult.results) {
+      console.log(`- Document ${result.id}`);
+      if (result.error) {
+        const { code, message } = result.error;
+        throw new Error(`Unexpected error (${code}): ${message}`);
+      }
+      console.log(`\tFHIR Bundle Returned: ${result.fhirBundle}`)
```

Comment:
Can we move this after printing the entities, I think it will also need to be stringified and formatted, e.g.
```suggestion
      console.log("\tFHIR Bundle:")
      console.log(JSON.stringify(result.fhirBundle, undefined, 4));
```